packages/mds-compliance-batch-processor/package.json

```diff
@@ -23,6 +23,7 @@
     "@mds-core/mds-compliance-engine": "workspace:0.7.12",
     "@mds-core/mds-compliance-service": "workspace:0.10.6",
     "@mds-core/mds-db": "workspace:0.4.32",
```

Comment:
Do we have anything that'll alert us when we don't need mds-db anymore in a package?

Reply:
Nope